phys2bids/cli/run.py

```diff
@@ -22,136 +22,186 @@ def _get_parser():
     """
     parser = argparse.ArgumentParser()
```

Comment:
Some of the parser helps got weird. we should probably manually edit them after black - unless I missed the part in which black doesn't substitute `'` with `"` if `"` is contained in the string, but then it becomes incredibly ugly and confusing.

Reply:
I've gone through and cleaned up the strings. Some switches between `'` and `"` are necessary, although the internal double-quotes could be changed to single-quotes to eliminate them.

Reply:
Sounds reasonable! In fact, I think they were double quote for a similar reason (not to confuse them with the external single quotes).

Reply:
Awesome. I've switched them.